setup.cfg

```diff
@@ -37,4 +37,4 @@ test_suite = tests
 universal = true
 
 [sdist]
-formats = zip, gztar
+formats = gztar
```

Comment:
Are we only leaving tarballs?

Reply:
@ffyuanda If two packages are generated at the same time, using semantic-release to upload to pypi will generate an error. I'm not sure if there is a better way to control uploading only one zip.

Reply:
Could you please paste the link to the error here? Guess it's somewhere in the Actions CI process?

Reply:
https://github.com/Steve0x2a/test-casdoor-python-ci/runs/5491755663?check_suite_focus=true 
![image](https://user-images.githubusercontent.com/17063270/157668421-5c5297e8-c778-44fe-92e4-c506ed2aed51.png)

Reply:
Alright, we can keep `gztar` as long as it's useful enough.